RelationshipMatrix/Converters/OrderedCategoryListConverter.cs

```diff
@@ -0,0 +1,68 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="OrderedCategoryListConverter.cs" company="RHEA System S.A.">
+//   Copyright (c) 2015-2019 RHEA System S.A.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4RelationshipMatrix.Converters
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Windows.Data;
+    using CDP4Common.SiteDirectoryData;
+
+    /// <summary>
+    /// The converter to convert a <see cref="List{Category}"/> to a sorted <see cref="List{Category}"/> by Name
+    /// </summary>
+    public class OrderedCategoryListConverter : IValueConverter
+    {
+        /// <summary>
+        /// The conversion method converts a <see cref="List{T}"/> of <see cref="Category"/> to a sorted <see cref="List{Category}"/> by Name.
+        /// </summary>
+        /// <param name="value">
+        /// The incoming value.
+        /// </param>
+        /// <param name="targetType">
+        /// The target type.
+        /// </param>
+        /// <param name="parameter">
+        /// The parameter passed on to this conversion.
+        /// </param>
+        /// <param name="culture">
+        /// The culture information.
+        /// </param>
+        /// <returns>
+        /// The <see cref="object"/> containing the same objects as the input collection.
+        /// </returns>
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return ((IList) value)?.Cast<Category>().OrderBy(c => c.Name).ToList() ?? new List<Category>();
```

Comment:
Maybe begin every next function on a new line. 
Makes it a bit more readable.

Reply:
When a line does not exceed certain length I actually prefer one-line, reads like a sentence for me.